flexmeasures/utils/unit_utils.py

```diff
@@ -20,3 +20,12 @@ def determine_stock_unit(flow_unit: str, time_unit: str = "h"):
         if flow_unit.endswith(f"/{time_unit}")
         else f"{flow_unit}{time_unit}"
     )
+
+
+def is_power_unit(unit: str) -> bool:
+    """For example:
+    >>> is_power_unit("kWh")  # True
+    >>> is_power_unit("°C")  # False
+    >>> is_power_unit("EUR/kWh")  # True
+    """
+    return unit in ("W", "Wh", "kW", "kWh", "MW", "MWh")
```

Comment:
Wh, kWh and MWh are not power units.

Reply:
That's true.

Would it not make sense to also include in the dashboard sensors which report energy (e.g. kWh)?

Reply:
Can you give an example?

Reply:
No. Metering is a big world. If you know about a general rule which lets all sensors send power data exclusively, let me know.

Reply:
> Would it not make sense to also include in the dashboard sensors which report energy (e.g. kWh)?

Then also, no.

Reply:
I can't parse this answer. Does it mean "no"?